### DISCUSSION TITLE
I have an error when I install whisper for my project

### DISCUSSION THREAD

**Discussion by NguyenMinhTri0209** [openai/whisper#1645] [Q&A] [answered]
PS D:\Python\TensorFlow> pip install git+https://github.com/openai/whisper.git
Collecting git+https://github.com/openai/whisper.git
  Cloning https://github.com/openai/whisper.git to c:\users\nminhtri2\appdata\local\temp\pip-req-build-w7avljik
  Running command git clone --filter=blob:none --quiet https://github.com/openai/whisper.git 'C:\Users\nminhtri2\AppData\Local\Temp\pip-req-build-w7avljik'
  fatal: unable to access 'https://github.com/openai/whisper.git/': error setting certificate verify locations:
    CAfile: cacert.pem
    CApath: none      
  error: subprocess-exited-with-error

  × git clone --filter=blob:none --quiet https://github.com/openai/whisper.git 'C:\Users\nminhtri2\AppData\Local\Temp\pip-req-build-w7avljik' did not run successfully.
  │ exit code: 128
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet https://github.com/openai/whisper.git 'C:\Users\nminhtri2\AppData\Local\Temp\pip-req-build-w7avljik' did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

**Comment by phineas-pta** [comment, marked as answer]
`pip install openai-whisper`

**Reply by NguyenMinhTri0209** [reply]
Oh, that's working, thanks so much!!!
It turns out it's that simple.

**Reply by Grace-Mercy22** [reply]
Thanks so much. This worked perfectly for me too.

**Reply by WesleyAldridge** [reply]
Doesn't work. Gives the same error message
